.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
I see. Well it's not mine either, I just make contributions here and there. But the 4 spaces seems to be more common anyway, so I don't mind, especially since there can easily be others who cringe at 2-space indentation :wink: Thanks for the nerdery links! Looks like some good reading.